### DISCUSSION TITLE
Need help - thousands of gantry-prune containers get created

### DISCUSSION THREAD

**Discussion by jpbaril** [shizunge/gantry#65] [General]
Hi,

I'm not totally shure what happened but in the last two weeks I got twice thousands of gantry-prune containers that were created and exited one after another. Today I deleted more than 11000 exited containers.

I have been using gantry for more than a year and it's the first time this is happening.

I know I had some issues with some swarm stacks and had to restart Docker around two weeks ago, so maybe that triggered that but I could not tell.

**Comment by shizunge** [comment]
I cannot debug you system. There are a few general advises: 1. exam the docker daemon logs. 2. change the [restart policy](https://docs.docker.com/reference/compose-file/deploy/#restart_policy)
